next/src/components/_global/cardsComponent/cardsComponent.tsx

```diff
@@ -0,0 +1,72 @@
+import styles from './cardsComponent.module.scss';
+import { type Cards } from '@/types/_global/Cards';
+import CentralizedHeading from '@/components/ui/centralizedHeading/centralizedHeading';
+import Img from '@/components/ui/Img';
+import Markdown from '@/components/ui/Markdown';
+
+export default function CardsComponent({ data: { centralizedHeading, cards } }: { data: Cards }) {
+  return (
+    <section className={styles.cardsComponent}>
+      <div className={` ${styles.componentWrapper} maxWidth`}>
+        <CentralizedHeading
+          data={centralizedHeading}
+          backgroundColor='var(--background)'
```

Comment:
Why you can't simply declare `backgroundColor` for `CentralizedHeading` inside the `.cardsComponent` in `./cardsComponent.module.scss`?

Reply:
This is a needed functionality for heading. If you are curious about its functionality you should check Heading component.

Reply:
This change should be made in dev branch, since there will be conflicts. It's a global change